### ISSUE TITLE
False-positive / incorrect warning: path segments do not use kebab-case

### ISSUE THREAD

**Issue by vaughany** [daveshanley/vacuum#308]
Hi there. In a newly created OpenAPI 3 spec file I have a path which ends in `.csv`:

```
paths:
  /reports/{reportName}.csv:
    get:
    ...
```

It contains the parameter key `{reportName}`:

```
  parameters:
    - name: reportName
      in: path
      description: name of report to be downloaded
      required: true
      schema:
        type: string
      example: 'some_kind_of_report'
```

Other similar paths with no extra text at the end do not complain about the same parameter key, however this 'csv download' path is flagged by Vacuum as a warning about the **case** of the parameter key:

```
Line / Column | Severity | Message                                                | Rule             | Path
(127:5)       | warning  | Path segments `{reportName}.csv` do not use kebab-case | paths-kebab-case | $.paths./reports/{reportName}.csv
```

...as I think it is inspecting the whole string `{reportName}.csv` with the `}` and the `.` and deeming it incorrect.

**Note:** I'm really new to OpenAPI specifications and Vacuum so I am making the assumption that having `.csv`, `.pdf` or similar on the end of an API path is 'legal' (I have certainly seen it done on the web and do it myself in several places) so sorry if this is not an error.

**Comment by daveshanley** [comment]
Yes, this is a bug and should be fixed, it's getting upset with the period and mistaking it for a failure, when it's not.

**Comment by daveshanley** [comment]
Fixed in `v0.3.7`
